mysql-test/suite/rocksdb_sys_vars/t/rocksdb_max_open_files_basic.test

```diff
@@ -1,6 +1,8 @@
 --source include/have_rocksdb.inc
 
---let $sys_var=ROCKSDB_MAX_OPEN_FILES
---let $read_only=1
---let $session=0
---source ../include/rocksdb_sys_var.inc
+# We can not use rocksdb_sys_var.inc here as this is a global, read only option
+# whose value is dependent on the servers open_files_limit. It is more fully
+# tested in the tocksdb.max_open_files test.
```

Comment:
`tocksdb` -> `rocksdb`

Reply:
@georgelorchpercona These changes look good. Could you update the pull request to fix the typo? We can commit it after.

Reply:
Rebased it to current fb-mysql-5.6.35 HEAD, fixed the typo and pushed. Thanks Herman!